easybuild/easyconfigs/m/MPICH/MPICH-3.2-GCC-4.9.3.eb

```diff
@@ -0,0 +1,19 @@
+name = 'MPICH'
+version = '3.2'
+
+homepage = 'http://www.mpich.org/'
+description = """MPICH v3.x is an open source high-performance MPI 3.0 implementation.
+It does not support InfiniBand (use MVAPICH2 with InfiniBand devices)."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.3'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.mpich.org/static/tarballs/%(version)s']
+
+sanity_check_paths = {
+    'files': ['bin/mpiexec', 'bin/mpirun',
+              'include/mpi.h', 'include/mpif.h'],
+    'dirs': [],
+}
```

Comment:
It is partly done by the easyblock. Not all the files are checked there, since not all the derivatives generate all the files. Example: PSMPI doesn't generate mpiexec and mpirun. The headers are currently not checked in the easyblock, but I guess it makes sense to add them.